netdeploy/networkTemplate.go

```diff
@@ -235,10 +235,14 @@ func (t NetworkTemplate) Validate() error {
 	}
 
 	// Follow nodes cannot be relays
+	// Relays cannot have peer list
 	for _, cfg := range t.Nodes {
```

Comment:
this might be true from the perspective of the tool, but there's nothing stopping a relay from having a phonebook in real operation, correct?

Reply:
correct